sle-release-notes-2025-01 (#240)
ni/install-systemlink-enterprise: release-notes/2025-01/README.md

# SystemLink Enterprise release 2025-01 Release Notes

The 2025-01 release for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.

## New Features and Behavior changes

> - The option to install MinIO as part of the top-level SLE Helm chart has been removed. MinIO was always intended to be used only during product evaluations where no other block storage solution was available. That option still exists, but the customer will need to install MinIO themselves rather than as part of the SLE application.

Reviewer: ```suggestion
-You can no longer install MinIO as part of the top-level SLE Helm chart. MinIO should be used only during product evaluations where no other block storage solution are available. You can still install MinIO on your own but not as part of the SLE application.
```

    - [Refer to the installation and upgrade guide on github](https://github.com/ni/install-systemlink-enterprise/tree/main/getting-started/templates/Dependencies/MinIO)
- Renamed the **Download files** privileges to **Preview and download files**.

> - Cross origin requests are enabled by default in the Dataframe service and the list of origins can be controlled in the top level helm chart.

Reviewer: ```suggestion
- Cross origin requests are enabled by default in the Data Frame service. You can control the list of origins in the top-level Helm chart.
```

> - Exposed an API to Cancel Executions. Removed **Cancel notebook executions** privileges to Data Maintainer role.

Reviewer: ```suggestion
- Exposed an API to Cancel Executions. Removed **Cancel notebook executions** privileges from the Data Maintainer role.
```

> - Software installation and feed configuration on multiple systems is now available.

Reviewer: ```suggestion
- Install software and configure feeds on multiple systems.
```

- Dynamic form fields (DFF)

>     - Made DFF privileges visible in rules management. The rights to manage DFF configuration are assigned to Data maintainer, Systems maintainer.

Reviewer: ```suggestion
    - Made DFF privileges visible in rules management. The rights to manage DFF configuration are assigned to the Data maintainer and Systems maintainer roles.
```
